docs/consensus.md

```diff
@@ -0,0 +1,46 @@
+# NEO-GO consensus node
+
+Neo-go node can act as a consensus node.
+To make this possible go dBFT implementation from [nspcc-dev/dbft](https://github.com/nspcc-dev/dbft) is used.
+
+## How to start your own privnet with neo-go nodes
+### Using existing Dockerfile
+This option assumes you have `docker` and `docker-compose` installed.
```

Comment:
I don't think an option can assume anything, so maybe it's better to rephrase it as prerequisites.